app/assets/stylesheets/overrides.scss

```diff
@@ -10,12 +11,34 @@ $navactive: $orange;
 $tabdown: $cream;
 $font: $brown;
 
+.container {
+background-color: #fff;
+border-radius: 20px ;
+}
+
+.container:not(nav-top), .container:not(nav-bottom) {
+  margin: 10px 30px;
+}
+
+.main-title {
+  padding: 5px 10px;
+  margin: 0;
+  background-color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.